vue.config.js

```diff
@@ -16,4 +16,9 @@ module.exports = {
   configureWebpack: {
     devtool: "source-map"
   },
+  devServer: {
+    host: '127.0.0.1',
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/Supershift/Sushi.Mediakiwi.UI/pullRequest?prid=8560396)
```suggestion
    host: "127.0.0.1",
```